templates/fine-tune-llm_v2/cookbooks/run_llmforge_as_anyscale_jobs/README.ipynb

```diff
@@ -0,0 +1,104 @@
+     "text": [
+      "name: \"llmforge-job\"\n",
+      "entrypoint: \"llmforge anyscale finetune config/llama-3-8b.yaml\"\n",
+      "image_uri: localhost:5555/anyscale/llm-forge:0.5.0.1-ngmM6BdcEdhWo0nvedP7janPLKS9Cdz2\n",
+      "requirements: []\n",
+      "max_retries: 0\n",
+      "working_dir: \".\""
+     ]
+    }
+   ],
+   "source": [
+    "!cat ./job_configs/job.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A few notes:\n",
+    "\n",
+    "- `entrypoint` is basically the command we want to run. Pay attention to the relative file location (config/llama-3-8b.yaml) and the `working_dir`. Inside `llama-3-8b.yaml` we are also referencing a relative path to `config/zero_3_offload_optim+param.json` which works because of specifying the `working_dir` to be the current dir `.`.\n",
+    "- `image_uri` refers to the image that has llmforge installed. On a the finetuning template the latest released image is automatically listed. For the full list of versions and their uris visit [llmforge versions](../../README.md#llmforge-versions). If you run this job from a workspace the image_uri will be inherited from the workspace image.\n",
+    "- `max_retries`: setting this to zero makes sure we do not keep retrying if the job fails. We should retry only when the job is flaky (maybe due to resource constraints, etc.)\n",
+    "- `working_dir`: Setting working_dir to current dir is necessary when you are submitting this job from outside of workspace (Your laptop or CI/CD pipelines)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1m\u001b[36mOutput\u001b[0m\u001b[0m\n",
+      "\u001b[0m\u001b[1m\u001b[36m(anyscale +0.8s)\u001b[0m \u001b[0m\u001b[0m\u001b[0m\u001b[0mSubmitting job with config JobConfig(name='llmforge-job', image_uri='localhost:5555/anyscale/llm-forge:0.5.0.1-ngmM6BdcEdhWo0nvedP7janPLKS9Cdz2', compute_config=None, env_vars=None, py_modules=None, cloud=None, project=None, ray_version=None).\u001b[0m\n",
+      "\u001b[0m\u001b[1m\u001b[36m(anyscale +7.0s)\u001b[0m \u001b[0m\u001b[0m\u001b[0m\u001b[0mUploading local dir '.' to cloud storage.\u001b[0m\n",
+      "\u001b[0m\u001b[1m\u001b[36m(anyscale +8.8s)\u001b[0m \u001b[0m\u001b[0m\u001b[0m\u001b[0mJob 'llmforge-job' submitted, ID: 'prodjob_dd6654glhg3h6t6sulq9y9c6aw'.\u001b[0m\n",
+      "\u001b[0m\u001b[1m\u001b[36m(anyscale +8.8s)\u001b[0m \u001b[0m\u001b[0m\u001b[0m\u001b[0mView the job in the UI: https://console.anyscale.com/jobs/prodjob_dd6654glhg3h6t6sulq9y9c6aw\u001b[0m\n",
+      "\u001b[0m\u001b[1m\u001b[36m(anyscale +8.8s)\u001b[0m \u001b[0m\u001b[0m\u001b[0m\u001b[0mUse `--wait` to wait for the job to run and stream logs.\u001b[0m\n",
+      "\u001b[0m\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!anyscale job submit --config-file ./job_configs/job.yaml"
```

Comment:
(I might not have gotten this right, I'm  following: https://docs.anyscale.com/reference/job-api/#job-cli )

```suggestion
    "!anyscale job submit --env WANDB_API_KEY=$WANDB_API_KEY --config-file ./job_configs/job.yaml"
```

Reply:
added a small hint doc at the bottom of the cookbook.